Mathlib/Data/Int/Order/Lemmas.lean

```diff
@@ -31,4 +31,100 @@ theorem natAbs_le_iff_mul_self_le {a b : ℤ} : a.natAbs ≤ b.natAbs ↔ a * a
   rw [← abs_le_iff_mul_self_le, abs_eq_natAbs, abs_eq_natAbs]
   exact Int.ofNat_le.symm
 
+theorem div_le_iff_of_dvd_of_pos (a b c : ℤ) (h1 : 0 < b) (h2 : b ∣ a) : a / b ≤ c ↔ a ≤ b * c := by
```

Comment:
Could you please perform the same changes as on your other PR about `Nat`? Eg here this should become
```suggestion
theorem div_le_iff_of_dvd_of_pos {a b c : ℤ} (hb : 0 < b) (hba : b ∣ a) : a / b ≤ c ↔ a ≤ b * c := by
```
but the other lemmas need changing too

Reply:
Done.

Reply:
Can you also fix the variable implicitness? Thanks!

Reply:
I followed the same approach as in #23100: the variables that appear in hypothesis are implicit. Those that don't, are explicit after the hypothesis.